.gitignore

```diff
@@ -1,8 +1,23 @@
-/.bundle/
-/.yardoc
-/_yardoc/
-/coverage/
-/doc/
-/pkg/
-/spec/reports/
-/tmp/
+*.gem
```

Comment:
I've included a .gitignore that i've been compiling for a long time now. It is quite complete. =)